client/src/App.js

```diff
@@ -139,6 +147,7 @@ class App extends Component {
     this.state = {
       loggedIn: false,
       fetchingAuth: true,
+      verified: Boolean,
```

Comment:
Won't this typescript notation throw an error?

Reply:
It doesn’t, but I can set it to false by default after changing its name